### PR TITLE
*: refine syncer check list

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

add more check list into syncer-support.md.

### What is changed and how it works?

update the support doc.

### Check List 

Tests 

 - No code

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan PTAL

File: .github/ISSUE_TEMPLATE/syncer-support.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ about: "Requesting support for syncer Tool"
 
 ## syncer Support
 
+Before submitting your issue, please check the requirements list below (all need to be met):
```
Comment (IANTHEREAL) [inline .github/ISSUE_TEMPLATE/syncer-support.md:9]: we can add most common check item in this issue template， and put others into a more detail issue template

Comment (csuzhangxc) [reply .github/ISSUE_TEMPLATE/syncer-support.md:9]: yes, but we have not implement this.
so, remove these?

Comment (IANTHEREAL) [reply .github/ISSUE_TEMPLATE/syncer-support.md:9]: we can create a more detail issue template/checklist template for debug deep into debug

**Pull request merged**